packages/apex-node/src/tests/asyncTests.ts

```diff
@@ -157,7 +158,7 @@ export class AsyncTests {
       message: nls.localize('retrievingTestRunSummary')
     });
 
-    const testRunSummaryResults = (await this.connection.tooling.autoFetchQuery(
+    const testRunSummaryResults = (await this.connection.tooling.query(
```

Comment:
you might want to pass `autoFetch: true` and `maxFetch` if you expect this to exceed the limit record of numbers returned per query

Reply:
👍  added authFetch.